README.md

```diff
@@ -1 +1,80 @@
-# Hello_Microverse_Project
\ No newline at end of file
+# Hello_Microverse_Project
+
+This project is learning setup for microverse student on how
+
+- Create branches with different base branches.
+- Follow Gitflow.
+- Understand the concept of version control.
+- Open pull requests in GitHub.
+- Write descriptive, easy to understand commit messages.
+- Write short, detailed, easy to understand descriptions of pull requests.
+- Write a descriptive and easy to understand README in business English for every project submitted.
+- Demonstrate an ability to submit a project for code review.
+
+![screenshot of the site](/img/screen_shot.PNG)
+
+In this project, you will set up a "Hello world" repository
+
+- Added three files that include html file, css file and test.md file
+- Set up linters files for Html and css
+  And I have made sure that the last file is ignored by gitignore
+
+# BUIT WITH
+
+- HTML
+- CSS
+- GIT
+
+# LIVE DEMO
+
+# GETTING STARTED
+
+- You can either use the github to access site both files locally (code) and the site created (By using the live demo link)
+
+  To get a local copy up and running follow these simple example steps.
+
+# Prerequisites:
+
+- Computer
+- Internet
+
+# Setup:
+
+# Install
+
+- Install Web browser (firefox)
+- Install node latest version (from NPM)
+- Install code editor (visual studio)
+
+# Usage
+
+- clone the repo or download
+- open the files of the repo through code editor
+
+# Run tests
+
+# Deployment
+
+- Run the index file on server
+
+# 👤 Author1
+
+    GitHub: @ayebaishmo
+    Twitter: @ishmo256
```

Comment:
- [ ] **[OPTIONAL]** For quicker navigation to your profiles, it would probably be better if these were links rather than plain text. You can format README links like so:
```
[Github](https://github.com/ayebaishmo)
[Twitter](https://twitter.com/ishmo256)
```